src/main/java/uk/gov/hmcts/reform/sendletter/services/date/holidays/BankHolidaysClient.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.sendletter.services.date.holidays;
+
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.sendletter.services.date.holidays.response.Holidays;
+
+public class BankHolidaysClient {
+
+    private final RestTemplate restTemplate;
+    private final String url;
+
+    public BankHolidaysClient(RestTemplate restTemplate, String url) {
+        this.restTemplate = restTemplate;
+        this.url = url;
+    }
+
+    public Holidays getForEngland() {
```

Comment:
Is there anything England-specific here?

Reply:
good point, it used to append `/england-and-wales.json` to the 'root' url.
Will fix.

Reply:
fixed

Reply:
```suggestion
... or we could have this bit in the url in the config
```